alacritty_terminal/src/term/mod.rs

```diff
@@ -1527,14 +1527,21 @@ impl<T: EventListener> Handler for Term<T> {
             Attr::CancelItalic => cursor.template.flags.remove(Flags::ITALIC),
             Attr::Underline => {
                 cursor.template.flags.remove(Flags::DOUBLE_UNDERLINE);
+                cursor.template.flags.remove(Flags::UNDERCURL);
```

Comment:
Could combine all the remove statements. We could also add something like a predefined ALL_UNDERLINES flag that combines them in the bitflags automatically?